<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_titulo">
        <property name="font">
         <font>
          <pointsize>19</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sistema de Gestión de Torneo de Fútbol</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_equipos">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Gestión de Equipos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_participantes">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Gestión de Participantes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_partidos">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Gestión de Partidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_resultados">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Actualizar Resultados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clasificacion">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Clasificación/Eliminatorias</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_creditos">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Créditos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ayuda">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Ayuda</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exportar">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Exportar datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_salir">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
